test/Unosquare.Swan.Test/ObjectMapperTest.cs

```diff
@@ -135,5 +135,17 @@ public void MapInvalidOperationExceptionTest()
                 mapper.Map<UserDto>(_sourceUser, false);
             });            
         }
+        
+        [Test]
+        public void RemoveMapPropertyPropertyDestinationInfoNull()
```

Comment:
PropertyPropertyPropertyPropertyPropertyPropertyPropertyPropertyProperty

Reply:
@ArCiGoUnoSquare

Reply:
@AlexeyTurlapov The name was changed. I'm waiting for the engineer's approval